README.md

```diff
@@ -381,6 +381,12 @@ Memory networks are implemented in [MemNN](https://github.com/facebook/MemNN).
 
 * [DeepNLP-models-Pytorch](https://github.com/DSKSD/DeepNLP-models-Pytorch) has Pytorch implementations of various deep NLP models used in CS224n(Stanford) in the form of Jupyter notebooks.The models are aimed for those who are acquainted with Pytorch.
 
+## Multilingual NLP Frameworks
+
+[Back to Top](#contents)
+* [UDPipe](https://github.com/ufal/udpipe) : Trainable pipeline for tokenizing, tagging, lemmatizing and parsing Universal Treebanks and other CoNLL-U file
```

Comment:
Hey @tiberiu44 , thanks for the PR!

Have you used UDPipe personally? Can you vouch for it's performance and stability?

Reply:
You are right. I'll update the two descriptions and include Spacy after I get the review from my friend. Thanks.

Reply:
Thanks @tiberiu44 

We have Spacy in the README. If you'd like to add a specific comment/review of why you like it - we'd like that :)